.github/workflows/expire.yml

```diff
@@ -76,9 +76,17 @@ jobs:
               // Check if the PR is an AOD request by checking the AOD file existence.
               if (files.data.find(f => (f.filename === 'iam.yaml' || f.filename === 'tool.yaml'))) {
                 // Check if the PR is within the cutoff threshold
-                const updatedAt = new Date(Date.parse(pull['updated_at']));
+                // Get last committed at time from ref branch
+                const ref = pull['head']['ref']
+                const branch = await github.rest.repos.getBranch({
+                  owner: repoOwner,
+                  repo: repoName,
+                  branch: ref,
+                })
+                const lastCommittedAt = branch['data']['commit']['commit']['committer']['date']
+                core.info(`#${pull['number']} lastCommittedAt (${lastCommittedAt})`);
 
-                if (now.getTime() - updatedAt.getTime() < cutoffMs) {
+                if (now.getTime() - new Date(Date.parse(lastCommittedAt)).getTime() < cutoffMs) {
```

Comment:
Both time uses UTC, right?

Reply:
since they are both data object, the getTime() returns the same format: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime